hi-ml-histopathology/testhisto/testhisto/datamodules/test_slides_datamodule.py

```diff
@@ -24,17 +24,17 @@ def mock_wsi_root_dir(tmpdir_factory: pytest.TempdirFactory) -> py.path.local:
     tmp_root_dir = tmpdir_factory.mktemp("mock_wsi")
```

Comment:
Oh good point, I missed this part of the documentation. I chose TempdirFactory so that we create the mock data only once per session, but  TempPathFactory is definitely a better option, I had to alter the typing annotations in the other classes to support py.path.local consequently. Thanks for pointing it out!